snippets/cytof/visualization/example_flowanalysis.Rmd

```diff
@@ -0,0 +1,664 @@
+---
+title: "Example Workflow TSNE, Phenograph, Heatmap"
+author: "Vito Zanotelli, vito.zanotelli@gmail.com"
+date: "01/12/2016"
+output:
+  html_document
+---
+
+# Aim
+This script describes an example analysis for cyTOF data in R. 
+
+The base of this script was developed beginning of 2016 and only slighlty changed since. Thus it is likely that there are better ways to dot his nowadays!
+Alternative resources for cyTOF analysis are:
+
+  - https://f1000research.com/articles/6-748/v3
+  - http://cytof.biosurf.org/#!/cytof
+  - https://www.jimmunol.org/content/200/1/3
+  - https://github.com/lmweber/FlowSOM-Rtsne-example
+
+## Load dependencies
+
+```{r Install helper library}
+if(!require('bbRtools')){
+  devtools::install_git('https://github.com/BodenmillerGroup/bbRtools')
+}
+```
+
+If anything is not installed, install it via `install.packages`.
+For Bioconductor packages, follow the bioconductor instructions.
+```{r Libraries, include=FALSE}
+library(data.table)
+library(Rtsne)
+library(RColorBrewer)
+library(dplyr)
+library(dtplyr)
+library(gplots)
+library(cba)
+library(bbRtools)
+library(gplots)
+library(ggplot2)
+```
+
+
+
+## Settings
+Set your project specific folders and settings
+```{r Settings}
+# input files: paths to your input and output folder
+
+# a folder containing FCS files to be analysised
+fcs_folder = './exampledata/'
+out_folder = '/home/vitoz/Data/rseminaroutput/'
+
+# the random seed
+rand_seed = 1234
+# Should the script use only the cells subsampled at the tsne step?
+subsampled_cells_only = F
+
+## load tsne: should the script try to load an existing tsne?
+load_tsne = F
+load_phenograph = F
+load_flowsom = F
+
+# filename of the tsne saved by the script
+# choose the filename of the tsne such that it reflects the tsne setting
```

Comment:
`fn_tsne = file.path(out_folder, '20170222_tsne_out.rdata')`
`fn_phenograph = file.path(out_folder, '20170222_pheno_out.rdata')`
`fn_flowsom = file.path(out_folder, '20170222_flowsom_out.rdata')`

doesn't work for me to save objects with the path specified above (creates '//' file path separator)

could be changed to 

`fn_tsne = file.path(out_folder, '20170222_tsne_out.rdata', fsep='')`
`fn_phenograph = file.path(out_folder, '20170222_pheno_out.rdata', fsep='')`
`fn_flowsom = file.path(out_folder, '20170222_flowsom_out.rdata', fsep='')`

Reply:
As an alternative fix, I removed trailing / at all folders. Can you check if this fixed the issue?

Reply:
@toobiwankenobi Did this fix the issue?

Reply:
@votti yep, this solved the issue.. sorry for my late reply.